src/matrix.jl

```diff
@@ -160,25 +160,53 @@ The matrix ``p``-norm of an interval matrix ``A`` is defined as
 ```
 
 where ``\\max`` and ``|·|`` are taken elementwise.
-
-### Algorithm
-
-We exploit that
-
-```math
-    ‖A‖_p = ‖\\max(-\\text{inf}(A), \\text{sup}(A))‖_p
-```
-
-to avoid taking the absolute (``|·|``).
 """
 function LinearAlgebra.opnorm(A::IntervalMatrix, p::Real=Inf)
-    if p == Inf || p == 1
-        return LinearAlgebra.opnorm(max.((-).(inf(A)), sup(A)), p)
+    if p == Inf
+        return _opnorm_inf(A)
+    elseif p == 1
+        return _opnorm_1(A)
     else
         error("the interval matrix norm for this value of p=$p is not implemented")
     end
 end
 
+# The operator norm in the infinity norm corresponds to the
+# maximum absolute value row-sum.
+function _opnorm_inf(A::IntervalMatrix{T}) where {T}
+    m, n = size(A)
+    res = zero(T)
+    @inbounds @simd for i in 1:m
+        acc = zero(T)
+        for j in 1:n
```

Comment:
I think you confused `i` and `j` here.

Reply:
The inf norm is the row-sum: fix the row `i` and sum over columns `j`. The previous version was using the matrix transpose, that is why they had to be swapped.

Reply:
Usually there is `for i in 1:n` and `for j in 1:m`.

Reply:
Here `m, n = size(A)`, `i` runs through the rows, and `j` through the columns.

Reply:
The indices in `_opnorm_1` were swapped, perhaps that was the source of the confusion.

Reply:
Argh, so it was [that comment](https://github.com/JuliaReach/IntervalMatrices.jl/pull/136#discussion_r378219650)...